source/acquaman/AMStepScanActionController.cpp

```diff
@@ -118,13 +120,14 @@ void AMStepScanActionController::buildScanController()
 
 	QFileInfo fullPath(AMUserSettings::defaultRelativePathForScan(QDateTime::currentDateTime()));	// ex: 2010/09/Mon_03_12_24_48_0000   (Relative, and with no extension)
 
+	scan_->addAMDSFilePath(fullPath.filePath());
```

Comment:
Is there an issue with setting the AMDSFilePath even if a beamline doesn't have an AMDS?